sdk/python/tests/integration/feature_repos/universal/data_sources/file.py

```diff
@@ -39,9 +39,10 @@ def create_data_source(
         self,
         df: pd.DataFrame,
         destination_name: str,
-        timestamp_field="ts",
+        event_timestamp_column="ts",
```

Comment:
`event_timestamp_column` is deprecated isn't it?

Reply:
I added it for passing the tests, although it is not used. Let me see if I can remove it.